src/main/java/org/terasology/climateConditions/FrostbiteSystem.java

```diff
@@ -67,18 +67,18 @@ public void beforeRemoveHypothermia(BeforeRemoveComponent event, EntityRef playe
         delayManager.cancelPeriodicAction(player, FROSTBITE_DAMAGE_ACTION_ID);
     }
 
-    @ReceiveEvent(components = {HypothermiaComponent.class})
-    public void onPeriodicFrostbite(PeriodicActionTriggeredEvent event, EntityRef player) {
+    @ReceiveEvent
+    public void onPeriodicFrostbite(PeriodicActionTriggeredEvent event, EntityRef player, HypothermiaComponent hypothermia) {
         if (event.getActionId().equals(FROSTBITE_DAMAGE_ACTION_ID)) {
-            applyFrostbiteDamagePlayer(player);
+            applyFrostbiteDamagePlayer(player, hypothermia);
```

Comment:
consider using the field instead of the component already here
```suggestion
            applyFrostbiteDamagePlayer(player, hypothermia.getAllEffectModifier());
```

Reply:
Ahh yes 😅, working on something like that now.

Reply:
I have updated it to have different effects on different levels. For now  I have still retained the allEffectModifier even though it isn't being used right now, in case we choose to use it..